src/EFCore.PG/Query/ExpressionTranslators/Internal/NpgsqlCompositeMethodCallTranslator.cs

```diff
@@ -55,19 +62,35 @@ public class NpgsqlCompositeMethodCallTranslator : RelationalCompositeMethodCall
             new NpgsqlRangeTranslator()
         };
 
+        /// <inheritdoc />
         public NpgsqlCompositeMethodCallTranslator(
             [NotNull] RelationalCompositeMethodCallTranslatorDependencies dependencies,
             [NotNull] INpgsqlOptions npgsqlOptions)
             : base(dependencies)
         {
+            var instanceTranslators =
+                new IMethodCallTranslator[]
+                {
+                    new NpgsqlDateAddTranslator(npgsqlOptions.PostgresVersion)
+                };
+
+            // ReSharper disable once DoNotCallOverridableMethodsInConstructor
+            AddTranslators(MethodCallTranslators);
+
             // ReSharper disable once DoNotCallOverridableMethodsInConstructor
-            AddTranslators(_methodCallTranslators);
+            AddTranslators(instanceTranslators);
 
             foreach (var plugin in npgsqlOptions.Plugins)
+            {
```

Comment:
I actually prefer leaving out curlies for simple one-liners, but I'm not die-hard on that.

Reply:
I removed this in the rebase. (I've omitted them elsewhere to match the style...not sure why I added this one.)